taxi/views.py

```diff
@@ -36,6 +43,21 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(ManufacturerListView, self).get_context_data(**kwargs)
+
+        context["manufacturer"] = SearchManufacturer()
```

Comment:
Use consistent naming in all your project. You used names `seach_model` and `search_username` in other forms, so rename this `manufacturer` key